docs/samples/plugins/navaid.rst

```diff
@@ -27,7 +27,7 @@ waypoint is more than 500 nm.
 Installation
 ~~~~~~~~~~~~
 
-1. Copy :download:`navaid.py <samples/plugins/navaid.py>` to a PYTHONPATH directory e.g. .config/mss
+1. Copy :download:`navaid.py </samples/plugins/navaid.py>` to a PYTHONPATH directory e.g. .config/mss
```

Comment:
intended?

Reply:
yes, this was broken, it didn't find the file. without the right location it tries ./samples/plugins/samples/plugins/navaid.py
That's the reason why we have only a download sign on https://mss.readthedocs.io/en/stable/samples/plugins/navaid.html but not a file connected